packages/ckeditor5/src/plugin.js

```diff
@@ -24,8 +24,8 @@ import StringManager from '@wiris/mathtype-html-integration-devkit/src/stringman
 import { MathTypeCommand, ChemTypeCommand } from './commands';
 import CKEditor5Integration from './integration';
 
-import mathIcon from '../theme/icons/formula.svg';
-import chemIcon from '../theme/icons/chem.svg';
+import mathIcon from '../theme/icons/ckeditor5-formula.svg';
+import chemIcon from '../theme/icons/ckeditor5-chem.svg';
```

Comment:
I'm missing an explanation of why do we have specific icons for CKEditor5, instead of using the same ones we use in the other demos. This is for future references, in case we bump into this part of the code and are unaware of why is this happening.

Reply:
For some reason the package `vite-plugin-ckeditor5` force us to add the prefix `ckeditor5-` on svg used on ckeditor5.